pkg/gatherers/clusterconfig/gather_cluster_ingress_certificates.go

```diff
@@ -0,0 +1,168 @@
+package clusterconfig
+
+import (
+	"context"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+
+	operatorclient "github.com/openshift/client-go/operator/clientset/versioned"
+
+	"k8s.io/klog/v2"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+
+	"github.com/openshift/insights-operator/pkg/record"
+)
+
+const ingressCertificatesLimits = 64
+
+var ingressNamespaces = []string{
+	"openshift-ingress-operator",
+	"openshift-ingress",
+}
+
+type CertificateInfo struct {
+	Name        string           `json:"name"`
+	Namespace   string           `json:"namespace"`
+	NotBefore   metav1.Time      `json:"not_before"`
+	NotAfter    metav1.Time      `json:"not_after"`
```

Comment:
I think there's no reason to define those as `metav1.Time`. We can simply have them as `time.Time`.